### DISCUSSION TITLE
W3U Playlist Support

### DISCUSSION THREAD

**Discussion by b0ogdan** [4gray/iptvnator#62] [Ideas]
Hi.
Thank you very much for your great work.
Awesome app and very happy that so many operating systems.
I recently discovered w3u playlist and the app wiseplay.
Not sure if it`s possible or easy to add support for this type of list into you app but that would be amazing.
Thank you

**Comment by 4gray** [comment]
Hi @b0ogdan thank you for the feedback!  

This is the first time I've heard of this format. As far as I understood w3u is a JSON file, so adding support for that would not be difficult. The only question is the popularity and relevance in the community.

Anyway, do you have an example of some legal playlist?

**Comment by b0ogdan** [comment]
Hi. I think the popularity is growing. It has some nice features. If you
have time take a look at wiseplay.. W3U it started with them from what i
understand.

Thank you very much for your hard work.
Looking forward for future updates 😚.

Have a great weekend.

**Comment by nunoacor** [comment]
Good job but I would like to make a suggestion regarding the ipTV player.
It should have the option in addition to html5 and video.js player to have the option to use external player or else the introduction of new player based on smplayer (mplayer). For me the best IPTV player for windows and linux. Thanks. NUNO
